app/components/SaleItems/saleItems.module.scss

```
@@ -0,0 +1,247 @@
+	left: 15px;
+	width: 96px;
+	height: 25px;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	background: rgb(31, 216, 93);
+	border-radius: 5px;
+}
+.popularText {
+	color: $color_3;
+	font-family: $font-family_1;
+	font-size: 10px;
+	font-weight: 600;
+	line-height: 150%;
+	text-align: center;
+	text-transform: uppercase;
+}
+.vectorBytilka {
+	position: absolute;
+	width: 9px;
+	height: 15px;
+	left: 25px;
+	top: 240px;
+	background: rgb(255, 255, 255);
+}
+.vectorBox {
+	position: absolute;
+	width: 20px;
+	height: 14px;
+	left: 23px;
+	top: 241px;
+	background: rgb(255, 255, 255);
+}
+.volumeText {
+	position: absolute;
+	top: 240px;
+	left: 52px;
+	width: 55px;
+	height: 16px;
+	color: $color_4;
+	font-family: $font-family_1;
+	font-size: 12px;
+	font-weight: 400;
+	line-height: 130%;
+	letter-spacing: 0%;
+	text-align: left;
+}
+.productDescription {
+	position: absolute;
+	width: 282px;
+	height: 42px;
+	left: 23px;
+	top: 266px;
+	color: $color_5;
+	font-family: $font-family_1;
+	font-size: 16px;
+	line-height: 130%;
+	text-align: left;
+	.bold {
+		font-weight: 500;
+	}
+	.normal {
+		font-weight: 400;
+	}
+}
+.barcode {
+	position: absolute;
+	width: 73px;
+	height: 21px;
+	color: $color_4;
+	font-family: $font-family_1;
+	font-size: 14px;
+	font-weight: 300;
+	text-align: left;
+	top: 332px;
+	left: 23px;
+}
+.barcodeNumber {
+	position: absolute;
+	color: $color_5;
+	font-family: $font-family_1;
+	font-size: 14px;
+	font-weight: 500;
+	text-align: left;
+	top: 2px;
+	left: 80px;
+}
+.producer {
+	position: absolute;
+	color: $color_4;
+	font-family: $font-family_1;
+	font-size: 14px;
+	font-weight: 300;
+	text-align: left;
+	top: 354px;
+	left: 23px;
+}
+.producerName {
+	color: $color_5;
+	font-family: $font-family_1;
+	font-size: 14px;
+	font-weight: 500;
+	margin-left: 5px;
+}
+.brand {
+	position: absolute;
+	color: $color_4;
+	font-family: $font-family_1;
+	font-size: 14px;
+	font-weight: 300;
+	text-align: left;
+	top: 374px;
+	left: 23px;
+}
+.brandName {
+	color: $color_5;
+	font-family: $font-family_1;
+	font-size: 14px;
+	font-weight: 500;
+	margin-left: 5px;
+}
+.price {
+	position: absolute;
+	width: 70px;
+	height: 21px;
+	left: 23px;
+	top: 430px;
+	color: $color_5;
+	font-family: $font-family_1;
+	font-size: 16px;
+	font-weight: 800;
+	line-height: 130%;
+	letter-spacing: 0%;
+	text-align: left;
+}
+@media (max-width: 768px) {
+	.saleItemsContainer {
+		width: 100%;
+		padding: 0 15px;
+	}
+	.title {
+		position: relative;
+		width: auto;
+		height: auto;
+		margin-top: 20px;
+		margin-left: 8px;
+		margin-right: 47px;
+		font-size: 20px;
+		font-weight: 500;
+		line-height: 120%;
+		text-align: left;
+		text-transform: uppercase;
+	}
+	.grid {
+		display: none;
+	}
+	.card {
+		display: none;
+	}
+	.image {
+		display: none;
+	}
+	.description {
+		display: none;
+	}
+	.popularTag {
+		display: none;
+	}
+	.volumeText {
+		display: none;
+	}
+	.productDescription {
+		display: none;
+	}
```

```suggestion
	.grid,
	.card,
	.image,
	.description,
	.popularTag,
	.volumeText,
	.productDescription {
		display: none;
	}
```

И точно ли необходимо скрывать столько элементов  отдельно?